app/models/employee.rb

```diff
@@ -20,5 +30,24 @@ def update_phone(value)
     self.save
   end
 
+  protected
+  def set_naming_format
+    self.first_name = self.first_name.downcase.capitalize
+    if self.middle_name!=nil
+      self.middle_name = self.middle_name.downcase.capitalize
+    end
+    self.last_name = self.last_name.downcase.capitalize
+  end
+
+  def confirm_deletion
+    puts 'Are you sure you want to delete'
+    input = gets
+    if(input=='y' || input=='Y')
```

Comment:
just other options of doing the same thing :wink:   
`['y','Y'].includes? input`  
input.downcase.eql? 'y'